test/controllers/ContentTest.scala

```diff
@@ -449,9 +449,11 @@ class ContentTest extends ControllerBaseSpec {
     "display form error message (date of marriage is after today’s date)" in {
       val localDate = LocalDate.now().plusYears(1)
```

Comment:
Do we need this line?
Looks like we are not using it.

Reply:
I hadn't spotted this. I'll use this value.